scripts/update-swagger-ui-statik.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
```

Comment:
When should this script be used? Do you mind adding some instructions?

Reply:
Anytime when we modify endpoints/related protocol buffer. I will make it more explicit in readme

Reply:
Done